services/ambari-server.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "ambari-server",
+  "description": "Ambari Server",
+  "dependencies": {
+    "conflicts": [],
+    "install": {
+      "requires": [ "base" ],
+      "uses": []
+    },
+    "provides": [],
+    "runtime": {
+      "requires": [],
+      "uses": []
+    }
+  },
```

Comment:
do you not need the config section with the ambari repos here like in the other server template?